tests/end_to_end/mt_kahypar_integration_tests.cc

```diff
@@ -124,52 +127,7 @@ using MultiLevelKm1Config = TestConfig<k,
                                        initial_partitioning_mode,
                                        LabelPropagationAlgorithm::label_propagation_km1>;
 
-template< PartitionID k,
-          bool use_community_detection,
-          InitialPartitioningMode initial_partitioning_mode>
-using MultiLevelCutConfig = TestConfig<k,
-                                       kahypar::Objective::cut,
-                                       use_community_detection,
-                                       initial_partitioning_mode,
-                                       LabelPropagationAlgorithm::label_propagation_cut>;
-
-typedef ::testing::Types<MultiLevelCutConfig<2,
-                                             false,
-                                             InitialPartitioningMode::recursive>,
-                         MultiLevelCutConfig<4,
-                                             false,
-                                             InitialPartitioningMode::recursive>,
-                         MultiLevelCutConfig<8,
-                                             false,
-                                             InitialPartitioningMode::recursive>,
-                          MultiLevelCutConfig<2,
-                                             false,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelCutConfig<4,
-                                             false,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelCutConfig<8,
-                                             false,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelCutConfig<2,
-                                             true,
-                                             InitialPartitioningMode::recursive>,
-                         MultiLevelCutConfig<4,
-                                             true,
-                                             InitialPartitioningMode::recursive>,
-                         MultiLevelCutConfig<8,
-                                             true,
-                                             InitialPartitioningMode::recursive>,
-                          MultiLevelCutConfig<2,
-                                             true,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelCutConfig<4,
-                                             true,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelCutConfig<8,
-                                             true,
-                                             InitialPartitioningMode::recursive_bisection>,
-                         MultiLevelKm1Config<2,
+typedef ::testing::Types<MultiLevelKm1Config<2,
```

Comment:
Good catch! In the improved_gain_recalculation there should be a commit that keeps these tests but deactivates FM refinement if cut objective is set. Might want to cherry pick that.